src/utilities/utilities.ts

```diff
@@ -173,3 +176,27 @@ export async function getXCTestPath(): Promise<string> {
     }
     return path.join(platformPath, "Developer", "Library", `XCTest-${version}`, "usr", "bin");
 }
+
+/**
+ * @returns SwiftPM flag for enabling test discovery
+ */
+export async function testDiscoveryFlag(ctx: FolderContext): Promise<string[]> {
```

Comment:
With this we have a couple of functions (see also win32BuildOptions) that are used to provide build options. They are in two different files. Perhaps we should centre them in one file BuildOptions.ts

Reply:
I moved it here because it is shared across multiple functionalities. If `win32BuildOptions` is also shared (currently not) I believe that’s worth consideration.

Reply:
I think a BuildOptions file is still worthwhile. The Windows options are probably going to get more complex if we look to add options for CodeView

Reply:
Since the flag is not necessary for `TestExplorer`, I’m going to move it back to `SwiftTaskProvider.ts` for this pitch. Maybe some day we’ll need `BuildOptions.ts` but I think that’s not now🤔 We only have ~10 lines and 2 functions now.